doc/tutorials/Examples12.rst

```diff
@@ -0,0 +1,52 @@
+.. _Examples12:
+
+Putting It All Together
+=========================
+
+Lets put together everything we have learned from the basic tutorials in the following examples : 
+
+
+.. Add the location of your Jupyter notebook below!
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+   :titlesonly:
+
+   qubit_rotation.ipynb
+   gaussian_transformation.ipynb
+   QGAN.ipynp
+
+
+.. Copy the template below in order to create a link to your notebook, and a thumbnail.
+
+.. _`Qubit Rotation`: qubit_rotation.html
+.. |qr| image:: figures/bloch.gif
+   :width: 260px
+   :align: middle
+   :target: qubit_rotation.html
+
+.. _`Gaussian Transformation`: gaussian_transformation.html
+.. |gt| image:: figures/bloch.gif
+   :width: 260px
+   :align: middle
+   :target: gaussian_transformation.html
+
+
+.. _`Quantum Generative Adversarial Networks`: QGAN.html
+.. |qgan| image:: figures/bloch.gif
+   :width: 260px
+   :align: middle
+   :target: QGAN.html
+
+.. Add your thumbnail to the table in the Gallery!
+
+.. rst-class:: gallery-table
+
++----------------------+--------------------------------+----------------------------------------------+
+| |qr|                 | |gt|                           | |qgan|                                       |
+|                      |                                |                                              |
+| `Qubit Rotation`_    | `Gaussian Transformation`_     | `Quantum Generative Adversarial Networks`_   |
```

Comment:
You might have said so, but just a reminder: the links are not working

Reply:
Which links are not working ? the thumbnails or the links in the jupyter files?

Reply:
These ones here: `Qubit Rotation`_  etc

Reply:
This is what renders the thumbnails. `Qubit Rotation`_ is the name that appears below that thumbnail. It connects to the html page corresponding to the handle |qr| which is given above the table. Do you have `nbsphinx` installed?

Reply:
Oh that's a possible source of error. I just did "make docs" as usual, and the link was broken.  Will it work if I install nbsphinx?

Reply:
yeah I hope so :)  I think it needs `nbsphinx` for  converting jupyter files to html files. you only need to install it. The `conf.py` file in the docs folder takes care of incorporating `nbsphinx` (in the latest commits at least :) )